README.md

```diff
@@ -138,20 +138,19 @@ Neither option is required, and the defaults will reduce the number of calls mad
 **Usage:**
 
 ```javascript
-const gptScript = require('@gptscript-ai/gptscript');
-
+const gptscript = require('@gptscript-ai/gptscript');
 
 const opts = {
     cache: false,
 };
 
-const prompt = `
-who was the president of the united states in 1928?
-`;
+const t = new gptscript.Tool({
+    instructions: "who was the president of the united states in 1928?"
+});
 
 async function streamExec() {
     try {
-        const { stdout, stderr, promise } = await gptScript.streamExec(prompt, opts);
+        const { stdout, stderr, promise } = await gptscript.streamExec(prompt, opts);
         if (stdout) {
```

Comment:
Why are stdout and stderr not set sometimes? (or if they always are, why are these wrapped in an if()?)